02-introducing-survey-data.Rmd

```diff
@@ -1,2 +1,188 @@
+
+Another crucial step in post survey processing is imputation. Often times there is item nonresponse where respondents do not answer specific questions. If the questions are crucial to analysis efforts or to the research question, researchers will implement imputation in an effort to reduce *item nonresponse error*. However, imputation itself can introduce *processing error* as well, so researchers should consider the overall implications of imputing data compared to having item nonresponse. There are multiple ways that imputation can be conducted, we recommend reviewing other resources like {cite Dever}<!--Stephanie is this the right book or is there a better one on imputation?--> for more information.
+
+#### Example: Number of Pets in a Household {- #overview-post-cleaning-ex}
+
+Let's return to the question we created in \@ref(overview-design-questionnaire-ex). The "other specify" invites respondents to specify the type of animal they prefer to have as a pet. If respondents entered answers such as "puppy", "turtle", "rabit", "rabbit", "bunny", "ant farm", "snake", "Mr. Purr", then researchers may wish to categorize these write-in responses to help with analysis. In this example, "puppy" could be assumed to be a reference to a `Dog`, and could be recoded there. The misspelling of "rabit" could be coded along with "rabbit" and "bunny" into a single category of `Bunny or Rabbit`. These are relatively standard decisions that a researcher could make. The remaining write-in responses could be categorized into a few different ways. "Mr. Purr", which may be someone's reference to their own cat, could be recoded to `Cat`, or it could remain as `Other` or some category that is `Unknown`. Depending on the number of responses related to each of others, they could all be combined into a single `Other` category, or maybe categories such as `Reptiles` or `Insects` could be created. Each of these decisions may impact the interpetation of the data, so our researcher should make sure to document the types of responses that fall into each of the new categories.
```

Comment:
Don't use reference and instead directly call out the "Number of Pets in a Household". The link will work now but will resolve to ?? since it isn't a numbered section